app/topics/shared/topic.service.ts

```diff
@@ -161,6 +161,20 @@ export class TopicService {
       );
   }
 
+  /**
+   * Get review status of supervisors
+   * @returns {Promise<Topic[]>} a Promise for a Topic[] object
+   */
+  public getReviewStatusOfCurrentUser(id: number) {
+    return this.cmsApiService.getUrl('/Api/Topics/' + id + '/ReviewStatus', {})
+      .toPromise()
```

Comment:
The API call returns all reviewers and the status for each review. So the name `getReviewStatusOfCurrentUser` doesn't really fit. Use `getReviews` etc. instead.